modules/tensor_mechanics/examples/umat_build/tests

```diff
@@ -0,0 +1,16 @@
+[Tests]
+  issues = '#18865'
+  design = 'AbaqusUMATStress.md'
+
+  [build]
+    type = 'RunCommand'
+    command = 'make'
+    requirement = "The multi-file UMAT build system shall successfully compile UMAT plugins."
```

Comment:
I'd drop "successfully" because that's kind of implied in every requirement.

Reply:
This is what tests `plugin.mk`